### PR TITLE
Highlight metrics exist for backup success

### PR DESCRIPTION
**Pull request by rcrowe**

### CODE REVIEW

File: docs/backup.md [hunk 1]
```diff
@@ -12,3 +12,15 @@ In order for the new backup schedule to get created in the db the backup job nee
...
+
+CockroachDB exports various Prometheus metrics, including the state of backup jobs. You can use these to alert you to the fact that a backup may not have successfully run for a period of time. 
+
+`jobs_backup_resume_completed` is increased by the node that successfully coordinated the backup for that run, so to make sure you are looking at the backup status across the cluster you would typically write the following query, which assumes you have 1 cluster for the namespace:
+
+```
+sum by (kubernetes_namespace) (increase(jobs_backup_resume_completed{kubernetes_namespace=~"auth|auth-customer"}[14h])) == 0
+```
+
+While the backup job may have successfully run, that is not a full proof way of confirming you'll be able to restore it 😅
```
Comment (matthewhughes-uw) [inline docs/backup.md:26]: out of curiosity: what are some of the stories here, is it possible to make a backup that's not actually usable?

Comment (rcrowe) [reply docs/backup.md:26]: @matthewhughes-uw For the most part I'm airing on the side of caution, those stories of people backing up and then coming to restore to find it doesn't work.

That if you want full faith in the system you need to automate & check your restore process. Perhaps the message is alarmist here?

**Review by matthewhughes-uw — APPROVED** [review]
Question: where does this metric come from? I see on https://www.cockroachlabs.com/docs/stable/backup-and-restore-monitoring (maybe worth linking) there is a metric `jobs.restore.resume_completed`

EDIT: I looked at an older version of those docs https://www.cockroachlabs.com/docs/v22.2/backup-and-restore-monitoring and see it, so I guess those metrics were renamed?

**Comment by MarcinGinszt** [comment]
I moved the PR [here](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/64)

**Pull request closed without merging**